drivers/i3c/i3c_shell.c

```diff
@@ -2390,8 +2396,8 @@ SHELL_STATIC_SUBCMD_SET_CREATE(
 		      cmd_i3c_info, 2, 1),
 	SHELL_CMD_ARG(speed, &dsub_i3c_device_name,
 		      "Set I3C device speed\n"
-		      "Usage: speed <device> <speed>",
-		      cmd_i3c_speed, 3, 0),
+		      "Usage: speed <device> <speed> <od high_ns> <od low_ns>",
```

Comment:
optional params need brackets around them in the help
```suggestion
		      "Usage: speed <device> <speed> [<od high_ns>] [<od low_ns>]",
```